unittest/bindings/test_force_cost.py

```diff
@@ -25,22 +29,16 @@
                                     ACTUATION.nu), 1.)
 MODEL = crocoddyl.DifferentialActionModelContactFwdDynamics(ROBOT_STATE, ACTUATION, CONTACTS, COSTS, 0., True)
 DATA = MODEL.createData()
-DATA.costs.costs["force"].contact = DATA.contacts.contacts["r_sole_contact"]
 
 # Created DAM numdiff
-MODEL_ND = crocoddyl.DifferentialActionModelNumDiff(MODEL, False)
+MODEL_ND = crocoddyl.DifferentialActionModelNumDiff(MODEL)
 dnum = MODEL_ND.createData()
-for d in dnum.data_x:
-    d.costs.costs["force"].contact = d.contacts.contacts["r_sole_contact"]
-for d in dnum.data_u:
-    d.costs.costs["force"].contact = d.contacts.contacts["r_sole_contact"]
-dnum.data_0.costs.costs["force"].contact = dnum.data_0.contacts.contacts["r_sole_contact"]
 
 x = ROBOT_STATE.rand()
 u = pinocchio.utils.rand(ACTUATION.nu)
 MODEL.calcDiff(DATA, x, u)
 MODEL_ND.calcDiff(dnum, x, u)
-np.allclose(DATA.Fx, dnum.Fx, atol=MODEL_ND.disturbance)
-np.allclose(DATA.Fu, dnum.Fu, atol=MODEL_ND.disturbance)
-np.allclose(DATA.Lx, dnum.Lx, atol=MODEL_ND.disturbance)
-np.allclose(DATA.Lu, dnum.Lu, atol=MODEL_ND.disturbance)
+assert (np.allclose(DATA.Fx, dnum.Fx, atol=1e5 * MODEL_ND.disturbance))
```

Comment:
I think this is a better option: https://docs.scipy.org/doc/numpy/reference/generated/numpy.testing.assert_allclose.html

Reply:
Also, 1e5?

Reply:
I need to work on this unittest (and test_momentum_cost) later. I won't spend much time now (i.e. this PR) in improving it.
Thanks for the feedback, I will consider it for the next PR.